### DISCUSSION TITLE
Prevent players from creating claims outside of predefined areas

### DISCUSSION THREAD

**Discussion by blenditbeautiful** [GriefPrevention/GriefPrevention#600] [Ideas]
As a server administrator I would like to be able to define regions in which players can create claims. 

This is because I would like to incentivize players to only build within certain areas while still giving them the freedom to roam, mine, etc in other parts of the map.

I understand I could achieve this by removing the ability of users to create claims and then assign subdivided claims but I don't want to manually assign each person a claim, instead I'd rather just pre-define an area in which they have permission to create claims.

I understand that this could be achieved with CreationRequiresWorldGuardBuildPermission but I do not want to prevent my players from building else where in the world, just incentivize them to keep builds they don't want to have griefed in certain areas.

Perhaps it could work like this:
1. (Config) EnableClaimRegions: True
2. /claimregions
3. "Claim regions mode. Players will be able to create claims here."
4. Use the golden shovel to create the claim region in the usual manner.
5. Non-admins can now claim land within the set claim region but not outside of it.

Apologies if I have missed something. Thanks for your time.

**Comment by TomLewis** [comment]
AreaShop?

**Comment by RoboMWM** [comment]
AreaShop won't be any different than manually creating subclaims, unless AreaShop now allows the player to define their own regions?
